src_code/src/controllers/services/Authentication/SignIn.js

```diff
@@ -13,7 +13,7 @@ const bcrypt = require('bcryptjs');
 async function SignIn(response, requestBody) {
     try {
         /** Validate sign in body to see if information they request to our endpoint is valid */
-        const errors = await SignInValidator(response, requestBody);
+        const errors = await SignInValidation(response, requestBody);
```

Comment:
nit: another redundant `await` here

Reply:
Await is not redundant. It should be used with resolved Promise. Async function returns promise but javascript does not ensure that promist is successfully resolve. I will update it.